lesson03/assignment/css/style.scss

```diff
@@ -4,3 +4,121 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+// responsive sizing for "screen and (max-width: 767px)"
+@include media-query('mobile') {
+  // border styling
+  .template-hook {
+    display: block;
+    border: 3rem solid $darkgray;
+    border-radius: 10px;
+    //margin: 1rem;
+    color: $darkergray;
+  }
+  // all card info
+  .card {
+    text-align: center;
+    img {
+      border-radius: 5%;
+      width: 50%;
+    }
+    h1 {
+      color: $darkergray;
+      margin-bottom: 0;
+    }
+    p {
+      font-style: italic;
+      font-size: 23px;
+      margin-top: 0;
+    }
+  }
+  .card-more {
+    background-color: $gray;
+    padding: 1rem 2rem;
+    font-size: 18px;
+    // hide the link underline
+    a:link {
+      text-decoration: none;
+    }
+    // show the link underline on hover
+    a:hover {
+      text-decoration: underline;
+    }
+  }
+}
+// responsive sizing for "screen and (min-width: 768px)"
+@include media-query('tab') {
+  // border styling
+  .template-hook {
+    display: flex;
```

Comment:
You'll want the cards to repeat downward, so if you're using flex you'll want to include: `flex-direction: column;`

Note this will create some other style issues to resolve. So you'll want to apply some responsive sizes (%, rem) to the images and other content that seems to fill the screen.